src/backend/gl/src/queue.rs

```diff
@@ -835,7 +996,27 @@ impl hal::queue::RawCommandQueue<Backend> for CommandQueue {
         S: 'a + Borrow<native::Semaphore>,
         Iw: IntoIterator<Item = &'a S>,
     {
+        let gl = &self.share.context;
+
         for swapchain in swapchains {
+            let extent = swapchain.0.borrow().extent;
+
+            gl.BindFramebuffer(gl::READ_FRAMEBUFFER, self.state.fbo);
```

Comment:
we won't come around blitting from the actual present image to the framebuffer in the future, but also fine for now. Should work for a large number of cases.